katas/content/nonlocal_games/ghz_quantum_strategy/solution.md

```diff
@@ -0,0 +1,12 @@
+In Q#, you can perform measurements in a specific basis using either the
+[Measure operation](https://docs.microsoft.com/qsharp/api/qsharp/microsoft.quantum.intrinsic.measure)
```

Comment:
The links to old APIs need to be updated to new

Reply:
Updated.